shared/routes/Guide.html

```diff
@@ -193,7 +195,67 @@
 <script>
 	import GuideContents from '../components/GuideContents.html';
 
+	function throttle ( fn, ms ) {
+		let blocked = false;
+
+		function unblock () {
+			blocked = false;
+		};
+
+		return function () {
+			if ( blocked ) return;
+			blocked = true;
+
+			fn();
+			setTimeout( unblock, ms );
```

Comment:
shouldn't do, why would it? The timeout is never meant to be cleared before it executes, and you obviously don't need to clear a timeout *after* it's been executed